packages/teams-js/src/private/files.ts

```diff
@@ -297,6 +297,28 @@ export namespace files {
     extension: string;
   }
 
+  /**
+   * Object used to represent a file
+   */
+  export interface File extends Blob {
+    /**
+     * Last modified timestamp
+     */
+    lastModified: Date;
+    /**
+     * Name of the file
+     */
+    name: string;
+    /**
+     * The file path.
+     */
+    webkitRelativePath: string;
```

Comment:
Why is this called the webkitRelativePath? I think our goal should be to have a platform-agnostic interface.

Reply:
Renamed the field to **filePath** and updated the comments to make it verbose.